test/test.js

```diff
@@ -162,13 +149,15 @@ try {
     if (sentry.length) logOnDetails.sha_sentryfile = sentry;
 }
 catch (beef){
-    util.log("Cannae load the sentry. " + beef);
+    util.log("Cannot load the sentry. " + beef);
```

Comment:
might want to rename this to example.js so we dinnae have merge conflicts with #147